### PR TITLE
Remove deprecated entries from security whitelistings

### PR DESCRIPTION
**Pull request by acipm**

### CODE REVIEW

**Comment by acipm** [comment]
@mgerstner

File: test/test_cli.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def test_parsing_config_file(test_arguments):
 def test_parsing_opensuse_conf(test_arguments):
     parsed = process_lint_args(test_arguments)
 
-    assert len(parsed['config']) == 13
+    assert len(parsed['config']) == 12
```
Comment (mgerstner) [inline test/test_cli.py:28]: Is it really a good idea to test for the exact number of entries of the production config? This test will surely fail every time somebody changes something in the configuration and cause a lot of noise and trouble.

Comment (acipm) [reply test/test_cli.py:28]: @marxin

Comment (marxin) [reply test/test_cli.py]: Yes, please remote it, it's pretty fragile as it seems.

**Comment by marxin** [comment]
Please rebase the branch so that CI is re-run.

**Comment by marxin** [comment]
@mgerstner Are you fine with the PR now?

**Comment by mgerstner** [comment]
> Please don't delete the device files whitelist, even if it is empty. We want to be able to quickly find the right file should we ever need to add a whitelisting there. Instead add a comment in there stating that currently no device files are whitelisted.

The file still is deleted, can you please adjust that?

**Comment by marxin** [comment]
> The file still is deleted, can you please adjust that?

The file is not deleted and there's a comment you requested.

**Comment by mgerstner** [comment]
> > The file still is deleted, can you please adjust that?
> 
> The file is not deleted and there's a comment you requested.

Actually is **gets** deleted but it is re-added in the follow-up commit. Well I can live with that.

**Comment by marxin** [comment]
Got it. @acipm Can you please squash the 2 commits?

**Pull request merged**